web/client/components/geostory/layouts/Cascade.jsx

```diff
@@ -114,6 +115,7 @@ const Cascade = ({
                 }
             </div>}
     </ContainerDimensions>
+    <WebPageCreator />
```

Comment:
@offtherailz is this ok?

Reply:
my opinion is to follow the same logic used for others components like the Text or Image Content 
i would change this and follow such a similar approach

Reply:
I don't think. Maybe is a good idea to render it in a Portal for the buttons interested to this (So you can even avoid the global state for web page editor and use the local state).
In any case, this breaks the structure (dependencies of specific contents should not go in general containers) so it should not be there. 

Not sure about how the UI should be, so this is a general comment. 
Generally take a look at how inline editor for map works. I think is a good Idea to use the same UI.

Reply:
As I specified in the issue description:

"As soon as the WebPage component is added to the story a modal opens to allow the user to specify the URL to embed."

so as fas as can I understand and @kboczkowski-clurgo showed to me, the current implementation seems good, at least regarding the UI part.